devlib/target.py

```diff
@@ -46,7 +46,8 @@
 from devlib.platform import Platform
 from devlib.exception import (DevlibTransientError, TargetStableError,
                               TargetNotRespondingError, TimeoutError,
-                              TargetTransientError, KernelConfigKeyError) # pylint: disable=redefined-builtin
+                              TargetTransientError, KernelConfigKeyError,
```

Comment:
`TargetTransientError` no longer necessary here as i s covered b `TargetError`

Reply:
The `TargetTransientError` is still raised explicitly in other parts of the code so the import is still required.